docs/content/references/cli/ptb.mdx

```diff
@@ -30,7 +30,9 @@ Options:
 
       --dry-run
           Perform a dry run of the PTB instead of executing it.
-          
+      --dev-inspect
+          Perform a dev-inspect of the PTB instead of executing it.
```

Comment:
```suggestion
          Perform a dev inspect of the PTB instead of executing it.
```
Maybe more consistent with the dry-run comment, just resolve if you don't feel like it